docs/wiki/deployment/aws-logging.md

```diff
@@ -11,6 +11,10 @@ Some configuration is shared between the two plugins:
 --aws_profile_name VALUE                AWS config profile to use for auth and region config
 --aws_region VALUE                      AWS region override
 --aws_secret_access_key VALUE           AWS secret access key override
+--aws_sts_arn_role VALUE                AWS STS assume role arn
+--aws_sts_region VALUE                  AWS STS assume role region
```

Comment:
I am fairly certain that if the aws region and the region for the sts assume role differ, this flag will be needed.  I can test it again.